README.md

```diff
@@ -21,16 +44,23 @@ path = /{path-to-spotify-heroic-datasource}
 If using Docker, you can mount this repo into the Grafana plugin directory:
 
 ```
-docker run -it -p 3000:3000 --name=grafana -v `pwd`:/var/lib/grafana/plugins/spotify-heroic-datasource grafana/grafana
+docker run -it -p 3000:3000 --name=grafana -v `pwd`/dist:/var/lib/grafana/plugins/spotify-heroic-datasource grafana/grafana
 ```
 
-Can also be installed with grafana-cli
 
-`grafana-cli plugins install spotify-heroic-datasource`
+# Releasing
+
+
+1. In a terminal, navigate to the repo (ensure you are on the master branch) and run:
+1. git pull --rebase origin master
+1. git log (ensure you are on the latest version)
+1. git fetch --tags
+1. git tag -l (this lists the previous heroic release tags. Increment the tag for your release. Use semantic versioning.
+1. git tag 1.0.1
```

Comment:
maybe make this {tag} instead of a specific version in the docs

Reply:
👍